api/turing/config/config.go

```diff
@@ -228,11 +228,12 @@ type SinglePageApplicationConfig struct {
 
 // DatabaseConfig config captures the Turing database config
 type DatabaseConfig struct {
-	Host     string `validate:"required"`
-	Port     int    `validate:"required"`
-	User     string `validate:"required"`
-	Password string `validate:"required"`
-	Database string `validate:"required"`
+	Host                       string `validate:"required"`
+	Port                       int    `validate:"required"`
+	User                       string `validate:"required"`
+	Password                   string `validate:"required"`
+	Database                   string `validate:"required"`
+	MigrationScriptsFolderPath string `validate:"required"`
```

Comment:
super nit, but it can be just `MigrationsPath`or `MigrationsFolder`

Reply:
Had a bit of a brain fart here. Yeah I'll do `MigrationsFolder` instead.